alphagov/paas-cf
# Remove pass-billing metrics from the platform metrics service

What
----

https://www.pivotaltracker.com/n/projects/1275640/stories/185341416

Removing paas-billing metrics from paas-metrics. We have new metrics that come from paas-billing itself. 

Why
----

This PR is a follow on from the [184604154](https://www.pivotaltracker.com/n/projects/1275640/stories/184604154), where paas-billing metrics were changed to be exported from the service itself, rather than scraping it's logit logs.

The graphs look okay apart from the request rate one which has been corrected as part of this PR.

How to review
-------------

- Deploy to a dev environment. Ensure there is only one set of billing dashboards now.
- Log into a dev credhub. Run `make upload-logit-secrets` to ensure logit credhub secrets are removed.
- Look at the code.

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
